examples/slack-clone/nextjs-slack-clone-dotenvx/README.md

```diff
@@ -116,6 +116,8 @@ Visit `localhost:3000` to test the app with GitHub OAuth integration.
 - **Vercel Account**
 - **Supabase Account**
 
+### How to Deploy Locally
```

Comment:
I just wanted to check this, as earlier in the doc there's a "local development" heading, and this suggestion is under the "remote deployment" heading, maybe a more generic "setup" or "configure environment variables" heading might work better?